### ISSUE TITLE
Quoted list identation

### ISSUE THREAD

**Issue by yenda** [kkinnear/zprint#175]
With this lint alias
```clojure
:lint
  {:extra-deps {clj-kondo/clj-kondo {:mvn/version "2020.12.12"}}
   :main-opts  ["-m" "clj-kondo.main"]}
```

Running zprint with no config
```bash
clojure -A:zprint -lsw test.clj
```

```clojure
(def ^:private config-keys
  '(bootstrapper cassandra graphql http-client measurer postgres redis service graphql http-client measurer postgres redis graphql http-client measurer postgres redis graphql http-client measurer postgres redis))
```

It idents a quoted list, which is not a function call as if the first element was a function
```clojure
(def ^:private config-keys
  '(bootstrapper cassandra
                 graphql
                 http-client
                 measurer
                 postgres
                 redis
                 service
                 graphql
                 http-client
                 measurer
                 postgres
                 redis
                 graphql
                 http-client
                 measurer
                 postgres
                 redis
                 graphql
                 http-client
                 measurer
                 postgres
                 redis))
```

**Comment by kkinnear** [comment]
Thanks for pointing this out!  A particularly good time, as I'm in the middle of some major implementation in exactly the area that this will need to change to do what you want.  

I assume that this is what you were expecting:
```clojure
(czprint i175 {:parse-string? true :fn-map {"bootstrapper" [:none {:list {:hang? false :indent 1}}]}})
(def ^:private config-keys
  '(bootstrapper
    cassandra
    graphql
    http-client
    measurer
    postgres
    redis
    service
    graphql
    http-client
    measurer
    postgres
    redis
    graphql
    http-client
    measurer
    postgres
    redis
    graphql
    http-client
    measurer
    postgres
    redis))
```
I am also thinking of extending the list printing so that it could be configured to work like the vector printing does now, that is, wrapping out to the current `:width` (which in this case is 80 characters):
```clojure 
(def ^:private config-keys
  '(bootstrapper cassandra graphql http-client measurer postgres redis service
    graphql http-client measurer postgres redis graphql http-client measurer
    postgres redis graphql http-client measurer postgres redis))
```
I'm currently unsure of what to do for a default for quoted lists.   It would be a change to make them all like the first example, above, but perhaps that could be considered a bug-fix, not an output change.  In either case, whether or not the first example is the default, getting the other behavior will be a very simple configuration change.

I expect to make the first example possible (and possibly the default) in the next release.  The second example also seems not terrible difficult (though I haven't yet actually tried it), so I'm thinking that I would like to offer that as an option as well.

As a temporary workaround to get the output to look more like what you would prefer, if you look at the first example, you can see something you could put in your `~/.zprintrc` file that would (totally as a hack, of course) cause this particular quoted list to come out the way that you see above.  This is what you would add to your `~/.zprintrc` file:
```clojure
 {:fn-map {"bootstrapper" [:none {:list {:hang? false :indent 1}}]}
```
 Of course, if you have an actual function called `bootstrapper`, this would be a bad idea, as it would cause all the calls to that function to look very odd.

**Comment by kkinnear** [comment]
This is now changed in `1.1.2`.

For example:
```
zprint.core=> (czprint i175 {:parse-string? true})
(def ^:private config-keys
  '(bootstrapper
    cassandra
    graphql
    http-client
    measurer
    postgres
    redis
    service
    graphql
    http-client
    measurer
    postgres
    redis
    graphql
    http-client
    measurer
    postgres
    redis
    graphql
    http-client
    measurer
    postgres
    redis))
nil
```
You can also get it to wrap, by doing this:
```
zprint.core=> (czprint i175 {:parse-string? true :fn-map {:quote :wrap}})
(def ^:private config-keys
  '(bootstrapper cassandra graphql http-client measurer postgres redis service
     graphql http-client measurer postgres redis graphql http-client measurer
     postgres redis graphql http-client measurer postgres redis))
nil
```
Unfortunately, I should have had `:quote` change the indent.  I will do that in the next release.  For now, if you want it to wrap, do this:
```
zprint.core=> (czprint i175 {:parse-string? true :fn-map {:quote [:wrap {:list {:indent 1}}]}})
(def ^:private config-keys
  '(bootstrapper cassandra graphql http-client measurer postgres redis service
    graphql http-client measurer postgres redis graphql http-client measurer
    postgres redis graphql http-client measurer postgres redis))
```
Thanks for bringing this up!

**Comment by kkinnear** [comment]
This is fixed in `1.2.0`.
